gen/build_deploy/aws.py

```diff
@@ -64,7 +61,8 @@ def validate_provider(provider):
         # template variable for the generating advanced template cloud configs
         'cloud_config': '{{ cloud_config }}',
         'rexray_config_preset': 'aws',
-        'fault_domain_detect_contents': calculate_fault_domain_detect_contents('aws')
+        'fault_domain_detect_contents': yaml.dump(
+            pkg_resources.resource_string('gen', 'fault-domain-detect/fd.sh').decode())
```

Comment:
Can we make this filename more descriptive and better reflect its intended use? Maybe `aws_gcp_azure.sh` or `cloud.sh` or something.